Templates/Files/MS SQL/MS SQL database SELECT to CSV file/ProcessExport.json

```diff
@@ -0,0 +1,66 @@
+﻿{
+	"Processes": [
```

Comment:
Also, the whole template folder (with all files) should be placed in "Databases/MicrosoftSQL" subfolder instead of "Files/MS SQL"